frontend/src/pages/Sessions/SessionsFeedV3/SessionsFeedV3.tsx

```diff
@@ -331,25 +300,12 @@ export const SessionFeedV3 = React.memo(() => {
 						<>
 							<OverageCard productType={ProductType.Sessions} />
 							{searchResultsCount === 0 ? (
-								showStarredSessions ? (
-									<SearchEmptyState
-										item="sessions"
-										customTitle="Your project doesn't have starred sessions."
-										customDescription={
-											'Starring a session is like bookmarking a website. ' +
-											'It gives you a way to tag a session that you want to look at again. ' +
-											'You can star a session by clicking the star icon next to the user details ' +
-											"in the session's right panel."
-										}
-									/>
-								) : (
```

Comment:
Do we not support starred sessions anymore? Wondering if we want to go another step in cleaning up starring logic in a future PR.

Reply:
yeah, we don't expose this in the UI anymore, I agree that there's probably more starring logic to clean up